diff --git a/node_modules/cbor/lib/constants.js b/node_modules/cbor/lib/constants.js
index 932e1c5..4b78a06 100644
--- a/node_modules/cbor/lib/constants.js
+++ b/node_modules/cbor/lib/constants.js
@@ -2,7 +2,7 @@
 
 // Let's get consistent first, then we can think about feature testing
 // for BigNumber support
-const {BigNumber} = require('bignumber.js')
+const {BigNumber} = global ? require('bignumber.js') : require('bignumber.js/bignumber.mjs')
 
 exports.BigNumber = BigNumber
 
