diff --git a/node_modules/cbor/lib/constants.js b/node_modules/cbor/lib/constants.js
index 932e1c5..9ca7538 100644
--- a/node_modules/cbor/lib/constants.js
+++ b/node_modules/cbor/lib/constants.js
@@ -2,7 +2,10 @@
 
 // Let's get consistent first, then we can think about feature testing
 // for BigNumber support
-const {BigNumber} = require('bignumber.js')
+const isNode = typeof process !== 'undefined'
+  && process.versions != null
+  && process.versions.node != null
+const {BigNumber} = isNode ? require('bignumber.js') : require('bignumber.js/bignumber.mjs')
 
 exports.BigNumber = BigNumber
 
